lib/nostrum/cache/channel_cache.ex

```diff
@@ -20,6 +20,15 @@ defmodule Nostrum.Cache.ChannelCache do
 
   import Nostrum.Snowflake, only: [is_snowflake: 1]
 
+  # dialyzer seems to think lookup will never fail,
```

Comment:
I think I want to look into this. If we're going to take care of these lets not add any more debt to the pile!